### PR TITLE
change(unlock-app): Allow claim always even if user has balance on arbitrum

### PR DESCRIPTION
**Pull request by searchableguy**
# Description

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by julien51 — COMMENTED** [review]
Lets do this only on arbitrum please. Otherwise it will cost us a ton.

File: unlock-app/src/components/content/event/EventDetails.tsx [hunk 1]
```diff
@@ -96,6 +96,8 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
       [lockAddress]: {
         network,
         emailRequired: true,
+        // TODO: Allow user to configure this?
+        forceClaim: true,
```
Comment (julien51) [inline unlock-app/src/components/content/event/EventDetails.tsx:100]: No no we need to hard code for now

Comment (julien51) [reply unlock-app/src/components/content/event/EventDetails.tsx:100]: Otherwise this will cost us a ton.

Comment (searchableguy) [reply unlock-app/src/components/content/event/EventDetails.tsx]: hardcoded. :D

**Pull request merged**